src/java/install.sh

```diff
@@ -206,15 +206,21 @@ sdk_install() {
     local full_version_check=${5:-".*-[a-z]+"}
     local set_as_default=${6:-"true"}
     if [ "${requested_version}" = "none" ]; then return; fi
-     # Blank will install default version SDKMAN has
     if [ "${requested_version}" = "default" ]; then
          requested_version=""
     elif echo "${requested_version}" | grep -oE "${full_version_check}" > /dev/null 2>&1; then
         echo "${requested_version}"
     else
         local regex="${prefix}\\K[0-9]+\\.?[0-9]*\\.?[0-9]*${suffix}"
         local version_list=$(su ${USERNAME} -c ". \${SDKMAN_DIR}/bin/sdkman-init.sh && sdk list ${install_type} 2>&1 | grep -oP \"${regex}\" | tr -d ' ' | sort -rV")
-        if [ "${requested_version}" = "latest" ] || [ "${requested_version}" = "current" ] || [ "${requested_version}" = "lts" ]; then
+        if [ "${requested_version}" = "lts" ]; then
+            check_packages jq
+            all_lts_versions=$(curl -s https://api.adoptopenjdk.net/v3/info/available_releases)
```

Comment:
I like the approach of how you are finding the LTS version 🎉 

<img width="1088" alt="image" src="https://github.com/user-attachments/assets/aafd7a47-dab8-4dab-add4-59ea0acae098">


Quick question, who hosts this API? Is it https://adoptium.net/ or someone else?
We need to ensure it is safe to use before we use this approach

Reply:
[The API at api.adoptopenjdk.net is hosted by AdoptOpenJDK, which has now moved to the Eclipse Foundation](https://api.adoptopenjdk.net/)[1](https://api.adoptopenjdk.net/). [As part of this transition, AdoptOpenJDK has rebranded to Adoptium, and it’s recommended to use the new API endpoint at api.adoptium.net](https://api.adoptopenjdk.net/)

Reply:
Great, thanks!